### ISSUE TITLE
WLASL Training Results Are Lower Than the Metrics Reported in the Paper

### ISSUE THREAD

**Issue by WaterLikely** [ZechengLi19/Uni-Sign#37]
Hi,

I am trying to reproduce the reported results on YouTube-ASL and WLASL, but the WLASL results I obtained are approximately 5 points lower than the results reported in the paper.

For the YouTube-ASL pre-training, I obtained the following results:

{
    "train_lr": 0.00028856593249478604,
    "train_loss": 3.8622053781799686,
    "test_loss": 5.34196981897249,
    "test_bleu1": 31.31993924204611,
    "test_bleu2": 21.00304084330353,
    "test_bleu3": 15.773605152658824,
    "test_bleu4": 12.459579299314473,
    "test_rouge": 25.660272077462327,
    "epoch": 7,
    "n_parameters": 587.747368
}

For WLASL, the best results I obtained are:

{
    "train_lr": 0.0002007973661644868,
    "train_loss": 3.0928707656774916,
    "test_loss": 3.735715653728294,
    "test_top1_acc_pi": 58.12053115423902,
    "test_top1_acc_pc": 56.105,
    "epoch": 19,
    "n_parameters": 587.747368
}

The WLASL results are approximately 5 percentage points lower than the results reported in the paper.

I have followed the official hyperparameter configuration as closely as possible. The experiments were conducted on 4 × NVIDIA L40 GPUs on a single machine.

Could you please help clarify what might be causing this performance gap?

In particular, I would like to know whether the difference could be related to:

The training or evaluation configuration;
The dataset preprocessing or data split;
The distributed training setup;
Differences in hardware or CUDA/PyTorch versions;
Any other settings that are not explicitly described in the paper.

Any suggestions on how to reproduce the reported WLASL results would be greatly appreciated.

Thank you!

**Comment by ZechengLi19** [comment]
@WaterLikely Sorry for the late reply. Could you share the performance curves from your fine-tuning runs (different ckpt)? For small datasets like this, changing the random seed can sometimes make a noticeable difference as well. Also, is the WLASL data you used the version I shared on Hugging Face?
